src/combination.jl

```diff
@@ -252,3 +250,32 @@ function plot_order_temporalCombin(params1, params2, g1s, g2s, named1, named2)
         title = string(named1, " --> ", named2, " - ", named2, " --> ", named1),
     )
 end
+
+####--------------- Loewe additivity ------------------####
+""" Find the inverse of a hill function (concentration), given the parameters and the effect. """
+function inv_hill(p::Array{Float64, 1}, y)
+    #p = [EC50, min, max, steepness], y:effect. it returns concentration
+    conc = p[1] * (((y - p[2]) / (p[3] - y)) ^ p[4])
```

Comment:
You're getting an error because the term being exponentiated is negative, which ends up creating a complex number. `y` can never be below `p[2]`, because that's the minimum of the function. Otherwise you're asking for the inverse of a value outside of the function's range, which isn't defined.

The minimum of both Hill curves might have to be shared because of this... it might help to see whether this is the case in other applications of Loewe.

Reply:
(p.s. Eid Mubarak!)

Reply:
Thank you, I will look into other papers that used Loewe.

Reply:
Thanks :)

Reply:
Oh yes, I remembered that Loewe is used in the case when the two drugs have the same minimum and maximum, and there is just a shift along the concentration axis. But I thought it is just theoretical! I still have to look at other papers.